modules/tinymce/src/themes/silver/test/ts/browser/window/SilverInlineDialogPositionTest.ts

```diff
@@ -119,6 +119,45 @@ describe('browser.tinymce.themes.silver.window.SilverInlineDialogPositionTest',
     });
   });
 
+  context('TINY-9554: dialog position with editor in fixed container', () => {
+    const setupElement = () => {
+      const container = SugarElement.fromHtml('<div style="position: fixed; top: 150; left: 150;"></div>');
```

Comment:
```suggestion
      const container = SugarElement.fromHtml('<div style="position: fixed; top: 150px; left: 150px;"></div>');
```
Without adding px units it is not actually fixed positioned

Reply:
- [ ] Fix pixel positions and also double check that test pass/fail when the you add/remove the fix

Reply:
ok, added the `px` and fixed the test, about trying add/remove the fix to check if the test is working I did it both previously and after this fix, and it works as expected.

If you got a different result, please lets check it together, it is possible that I'm missing something